dev-aws/kafka-shared/customer-billing.tf

```diff
@@ -23,8 +23,39 @@ module "invoice_fulfillment_producer" {
 module "bills_total_api_consumer" {
   source = "../../modules/consumer"
 
-  topic = kafka_topic.invoice_fulfillment.name
+  topic          = kafka_topic.invoice_fulfillment.name
   consumer_group = "bills-total-api-reader"
 
-  cert_common_name = "customer-billing/bills-total-api" 
+  cert_common_name = "customer-billing/bills-total-api"
+}
+
+resource "kafka_topic" "invoice_fulfillment_deadletter" {
+  name               = "bex.internal.accountreadytobefulfilled_deadletter"
+  replication_factor = 3
+  partitions         = 10
+  config = {
+    # keep data for 14 days
+    "retention.ms" = "1209600000"
+    # allow max 1 MB for a message
+    "max.message.bytes" = "1048576"
+    "compression.type"  = "zstd"
+    "cleanup.policy"    = "delete"
+  }
+}
+
+module "invoice_fulfillment_producer" {
+  source = "../../modules/producer"
+
+  topic = kafka_topic.invoice_fulfillment_deadletter.name
+
+  cert_common_name = "customer-billing/invoice-fulfillment"
+}
+
+module "invoice_fulfillment_consumer" {
```

Comment:
Can you delete the consumer for now. We will only produce and don't consume from the invoice-fulfillment service at the moment. If we want to keep it we have to change the consumer_group name and I don't know to what

Reply:
yes, In my head I was repeating "retries retries retries" hence why copied the consumer for no reason 🤦